modules/gathering-data-specific-features.adoc

```diff
@@ -78,6 +78,9 @@ endif::openshift-dedicated[]
 |`registry.redhat.io/openshift4/ose-csi-driver-shared-resource-mustgather-rhel8:v<installed_version_secret_store>`
 |Data collection for the {secrets-store-operator}.
 
+|`registry.redhat.io/lvms4/lvms-must-gather-rhel8:v<installed-version-LVMS>`
+|Data collection for the {lvms} (LVMS) Operator.
```

Comment:
Its either LVMS, or LVM Operator but not LVMS Operator

Reply:
Not quite:
The name of the operator is LVM Operator.
The name of the Product / Solution is LVMS. LVMS is an aggregate of LVM Operator and TopoLVM.

So if you refer to the whole solution, its LVMS, if you only refer to the operator its LVM Operator. Not the best naming, I agree...

Reply:
Thank you for the clarifying! :)
Updated it to LVM Operator.
Thanks.